app/services/repository_datatable_service.rb

```diff
@@ -119,15 +121,18 @@ def sort_rows(column_obj, records)
       end
     elsif sortable_columns[column_id - 1] == 'repository_cell.value'
       id = @mappings.key(column_id.to_s)
-      sorting_column = RepositoryColumn.find_by_id(id)
+      sorting_column = RepositoryColumn.find_by(id: id)
       return records unless sorting_column
 
       sorting_data_type = sorting_column.data_type.constantize
 
       cells = RepositoryCell.joins(sorting_data_type::SORTABLE_VALUE_INCLUDE)
                             .where('repository_cells.repository_column_id': sorting_column.id)
                             .select("repository_cells.repository_row_id,
-                                    #{sorting_data_type::SORTABLE_COLUMN_NAME} AS value")
+                                              STRING_AGG(
```

Comment:
It makes select query ~ 2 times slower

Reply:
Now we will use string aggregation only for checklists